fontforgeexe/fontview.c

```diff
@@ -6784,7 +6784,7 @@ static void FontView_ReformatAll(SplineFont *sf) {
     MetricsView *mvs;
     extern int use_freetype_to_rasterize_fv;
 
-    if ( sf->fv==NULL || ((FontView *) (sf->fv))->v==NULL || ((FontView *) (sf->fv))->colcnt==0 )			/* Can happen in scripts */
```

Comment:
Can you say more about the double check? If we're iterating through the list and checking `colcnt` each time below, why do we punt here if the first one is zero?

Reply:
I belive `colcnt` can be 0 if the fontview hasn't initialised yet.

If the script is called from the GUI, then the fontview may be in the process of initialising.

In that case we do still require to reinitialise fv->filled / fv->show to keep their views in sync, but we don't require updating fv->rowltot or re-exposing the fontview, hence moving the colcnt check to below.